proposals/stage0/REQUIREMENTS/README.md

```diff
@@ -21,13 +21,14 @@ The needs of projects and ecosystem stakeholders being delegated via the represe
 
 The requirements for the elected representatives are as follows:
 
-* They should be a regular member at the OpenJS Foundation.
-* They should be available to travel for the meetings.
-* They should be ready to handle the conflicts of interest that might arise.
-* They should focus / address the concerns from the OpenJS Foundation and its projects' prespective.
-* They should be update the community about the meetings (either present/write)
-* They should be responsible for making sure the agenda is ready before the meeting.
+* They should be a regular member at the OpenJS Foundation standards committee meetings.
```

Comment:
This changes the meaning of the first bullet point, which clearly should be clarified.

As far as I understand, the earlier intent was to require that the person be at least a [regular member of the CPC](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#regular-members), which is a well-defined role.

An alternative understanding of the requirement would be that the person is an "individual member" of the foundation, but that programme has not rolled out yet.

I would be very surprised if the actual intent was to require that the person regularly shows up to OpenJS Standards Team meetings.

Reply:
It should be just `regular member at the OpenJS Foundation`.

Reply:
Do you mean a regular member of the CPC, or something else? And if so, shouldn't voting members be also acceptable?

My recommendation for the phrasing would be "They should be a member of the OpenJS Cross Project Council."

Reply:
> They should be a member of the OpenJS Cross Project Council.

I am not sure whether there will be a scenario where someone will just be part of standards and not in CPC. That is the reason to mention to specify OpenJS Foundation. I think we can state `member at the OpenJS Foundation`.

Reply:
Thanks @eemeli @sendilkumarn 

Should I change to "should regularly attend OpenJS Foundation meetings"

I imagine that if they only attend standards meetings then they're less available to receive feedback and answer questions re standards than someone who attends other meetings.